### PR TITLE
Feature/global date format

### LINKED ISSUE #160: Use 'sv-SE' date format in field inputs
Use the Swedish date format "2012-12-20" in input TextFields and so on so we use the same date format.

### PR DESCRIPTION
**Pull request by norling**
Replaces all `<input type='date'>` controls with `material-ui/DatePicker`, and adds locale formatting to all printed date fields.

Closes #160

### CODE REVIEW

File: frontend/src/individual_edit.tsx [hunk 1]
```diff
@@ -229,208 +232,223 @@ export function IndividualEdit({id}: {id: string | undefined}) {
...
+              <Button variant="contained"
+                      color="primary"
+                      onClick={() => {updateField('weights', [...individual.weights, {date: weightDate, weight: weight}])}}>
+                {'Lägg till viktmätning'}
+              </Button>
+              <h3>Hull</h3>
+              <ul>
+                {individual.bodyfat && individual.bodyfat.map((b: DateBodyfat, i: number) =>
+                  <li key={i} className={style.measureList}>
+                    {`${new Date(b.date).toLocaleDateString(locale)} - ${b.bodyfat}`}
```
Comment (danr) [inline frontend/src/individual_edit.tsx]: Maybe just extract `d => new Date(d).toLocaleDateString(locale)` to a separate function

**Review by danr — APPROVED** [review]
Cool bananas :banana: 

I would introduce some helper functions for the repeated code

File: frontend/src/individual_edit.tsx (hunk 1 above)
Comment (norling) [reply frontend/src/individual_edit.tsx]: seems fair! :)

Comment (jhagberg) [reply frontend/src/individual_edit.tsx]: Yes this seems to work well when I test so perhaps address Dan's comments and its good to go

**Review by jhagberg — COMMENTED** [review]
Decide if you want to address Dans comments

**Comment by danr** [comment]
Nice! :clinking_glasses:

**Pull request merged**